src/main/java/com/elytradev/concrete/inventory/gui/ConcreteContainer.java

```diff
@@ -53,13 +53,13 @@
 	}
 	
 	private final IInventory playerInventory;
-	private final IInventory container;
+	private final IInventory inventory;
```

Comment:
while 'container' is an IInventory, it's more that it refers to the inventory of a block, entity, or item that we're running a gui for, so I'd almost prefer blockInventory or containerInventory *as distinct from playerInventory*